processing/src/main/java/org/apache/druid/query/Druids.java

```diff
@@ -869,6 +860,16 @@ public ScanQueryBuilder intervals(QuerySegmentSpec q)
       return this;
     }
 
+    /**
+     * Convenience method for an interval over all time.
+     */
+    public ScanQueryBuilder eternity()
```

Comment:
Would a name like `intervalEternity` or `withIntervalEternity` be more representative of what we are trying to do here?

The other methods are names of the params that the Scan query needs e.g. `dataSource`, `virtualColumns`, whereas `eternity` is a value of the param `intervals`.

Reply:
By popular demand, renamed this to `eternityInterval()`.